vars/jiraMinerPopulateReleaseIssues.groovy

```diff
@@ -0,0 +1,4 @@
+def call(version) {
+    sh """jira-miner populate "project in (RHMAP, FeedHenry, AeroGear, RAINCATCH) AND fixVersion in (${version}) and (status in ('Dev Complete', 'Ready for QA', Resolved, Done, Verified) or (status = Closed and resolution = Done)) and type in ('Feature Request', Bug, Improvement, Documentation,Enhancement,Clarification,'New Feature')"
```

Comment:
Should we try and make this generic since its a public shared library? Should we make these jira projects configurable?